More generic contact template

mdepasca-mqv opened the pull request:
Removed unnecessary input field from contact template.

The template opens a Request issue.

Commit:
🚧 Tailoring to major change request/collaboration

Commit:
🚧 Removing issues in template

Commit:
🚸 Add issue type

mdepasca-mqv committed:
🚧 Remove type field from contact_benchmarking.yml

Signed-off-by: Marco De Pascale <depascale@mailbox.org>

Commit:
🚸 More generic example email address

mnfarooqi reviewed (commented):
Were you able to test this new template? I am not able to create an issue in your fork to test it.

mnfarooqi commented:
I see this in your fork. 

<img width="909" height="195" alt="Screenshot 2025-12-19 at 13 31 45" src="https://github.com/user-attachments/assets/0363bf4a-93fb-469c-ae17-3488edb4c8ea" />

mdepasca-mqv committed:
Enhance contact benchmarking template with textarea

Added a textarea for request type in the contact benchmarking issue template.

Signed-off-by: Marco De Pascale <depascale@mailbox.org>

mdepasca-mqv committed:
Update contact benchmarking issue template

Changed textarea to dropdown for communication type selection with added options.

Signed-off-by: Marco De Pascale <depascale@mailbox.org>

mdepasca-mqv committed:
Change default option for communication type

Signed-off-by: Marco De Pascale <depascale@mailbox.org>

mdepasca-mqv committed:
Update contact_benchmarking.yml

Signed-off-by: Marco De Pascale <depascale@mailbox.org>

mdepasca-mqv committed:
Remove Communication Type dropdown from template

Removed the 'Communication Type' dropdown from the issue template.

Signed-off-by: Marco De Pascale <depascale@mailbox.org>

mdepasca-mqv committed:
Add communication type dropdown to issue template

Added a dropdown for selecting communication type in the issue template.

Signed-off-by: Marco De Pascale <depascale@mailbox.org>

mdepasca-mqv committed:
Update contact_benchmarking.yml

Signed-off-by: Marco De Pascale <depascale@mailbox.org>

mdepasca-mqv committed:
Merge branch 'Munich-Quantum-Software-Stack:develop' into develop
